internal/catalogmetadata/client/client_test.go

```diff
@@ -24,30 +23,30 @@ import (
 )
 
 func TestClient(t *testing.T) {
-	t.Run("Bundles", func(t *testing.T) {
```

Comment:
I removed this outer/unnecessary `t.Run` wrapper (mainly because it caused my IDE to not be able to find the individual test cases to run/debug separately).

In order to review changes in this test, I advise ignoring whitespace.